src/test/java/com/conveyal/datatools/manager/utils/sql/TableCheckTest.java

```diff
@@ -0,0 +1,59 @@
+package com.conveyal.datatools.manager.utils.sql;
+
+import com.conveyal.gtfs.loader.Table;
+import com.google.common.collect.Lists;
+import org.junit.jupiter.api.Test;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+class TableCheckTest {
+    private static final String NAMESPACE = "namespace";
+
+    @Test
+    void shouldDetectMissingTableColumns() {
+        List<String> removedColumns = Lists.newArrayList("route_id", "route_short_name");
+        List<ColumnCheck> columns = Arrays
+            .stream(Table.ROUTES.fields)
+            .filter(f -> !removedColumns.contains(f.name))
+            .map(ColumnCheck::new)
+            .collect(Collectors.toList());
+
+        TableCheck tableInfo = new TableCheck(Table.ROUTES, NAMESPACE, columns);
+        assertEquals(
+            String.join(",", removedColumns),
+            tableInfo.missingColumns.stream().map(c -> c.columnName).collect(Collectors.joining(","))
+        );
+    }
+
+    @Test
+    void shouldDetectColumnsWithWrongType() {
+        List<ColumnCheck> columns = Arrays
+            .stream(Table.ROUTES.fields)
+            .map(ColumnCheck::new)
+            .collect(Collectors.toList());
+
+        TableCheck tableInfo = new TableCheck(Table.ROUTES, NAMESPACE, columns);
+        assertTrue(tableInfo.columnsWithWrongType.isEmpty());
+
+        // Modify the type of one column
```

Comment:
Add fullstop.

Reply:
Fixed in https://github.com/ibi-group/datatools-server/pull/459/commits/a7f11ec7a8272a68de1a19d3ac498f0ca0ca78f1.